cluster-sync/nfs/nfs-server.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: nfs-data
+  namespace: cdi
+spec:
+  storageClassName: local
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 30Gi
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nfs-server
+  namespace: cdi
+  labels:
+    app: "nfs-server"
+spec:
+  containers:
+  - image: itsthenetwork/nfs-server-alpine:latest
```

Comment:
Do we want to select a specific version to avoid being unexpectedly broken?

Reply:
I can do a specific tag, but there is always a chance we will get broken unexpectedly due to changes to the tag. But let me change it to the specific tag.